pulser/parametrized/paramobj.py

```diff
@@ -36,16 +37,16 @@
 class OpSupport:
     """Methods for supporting operators on parametrized objects."""
 
-    def _do_op(self, op_name, other):
+    def _do_op(self, op_name: str, other: Union[int, float]) -> 'ParamObj':
```

Comment:
If you import `annotations` from `__future__` you can use the class instead of the string (eg. `ParamObj` instead of `'ParamObj'`). It's preferable to do this throughout the code